### ISSUE TITLE
forward-char at end of buffer

### ISSUE THREAD

**Issue by jrossie** [ahungry/org-jira#351]
https://github.com/ahungry/org-jira/blob/295b01ede42952c848bd8d76bc8c456a87876cbc/org-jira.el#L1175
Diagnosed an "End of buffer" error while running `org-jira-get-issues`.  When `org-jira--render-issue` attempts to render the description text it has two lines that generate an error in org 9.7.4:

```
                                                       (goto-char (point-max))
                                                       (org-insert-subheading t))
```
This is because `org-insert-subheading` immediately attempts `forward-char` when point is at beginning of line, which it is.  I fixed in my local copy by inserting a new line before calling `org-insert-subheading`, but I don't claim to understand the code well enough to submit as a PR.

```
                                                       (goto-char (point-max))
                                                       (open-line 1)
                                                       (org-insert-subheading t))
```

**Comment by ahungry** [comment]
Thanks @jrossie - looks like the same issue as https://github.com/ahungry/org-jira/issues/352 - do you know if this fix works with org 9.6 and 9.7, or just 9.7 and above?

I think we need to snapshot the org-insert-subheading function's original source code, if we can't rely on a consistent behavior between org versions.

**Comment by ahungry** [comment]
Thanks @jrossie - added in commit ac625b0
